composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "phpspec/phpspec": "^5.0"
+        "phpspec/phpspec": "^6.1"
```

Comment:
I think this should be `^5.0|^6.0`. Why drop 5? If we are going to drop 5, then we probably want another major release, like in the past.